code/modules/antagonists/changeling/abomination.dm

```diff
@@ -0,0 +1,102 @@
+/*
+	- CHANGELING ABOMINATION -
+
+	Contents:
+		Reagent
+		Monster
+*/
+
+/datum/reagent/shiftium
+	name = "shiftium"
+	description = "A strange stimulant."
+	color = "#9D5A99"
+	overdose_threshold = 10
+	taste_description = "savory with a bit of blood"
+	metabolization_rate = 0.3 * REAGENTS_METABOLISM
+	var/obj/shapeshift_holder/shapeshiftdata
+
+/datum/reagent/shiftium/on_mob_metabolize(mob/living/L)
+	L.adjustStaminaLoss(-5, 0)
+	..()
+
+/datum/reagent/shiftium/on_mob_end_metabolize(mob/living/L)
+	..()
+	if(!shapeshiftdata)
+		return
+	shapeshiftdata.restore()
+	REMOVE_TRAIT(L, CHANGELING_HIVEMIND_MUTE, type)
+
+/datum/reagent/shiftium/overdose_start(mob/living/L)
+	if (L.mind?.has_antag_datum(/datum/antagonist/changeling))
+		to_chat(L,"<span class='danger'>You struggle to maintain your form!</span>")
+	..()
+
+/datum/reagent/shiftium/overdose_process(mob/living/L)
+	L.adjustStaminaLoss(5, 0)
+	if (prob(volume/2))
```

Comment:
This is a pretty small chance.

Reply:
It can also pass multiple times and run a lot of redundant code.

Reply:
```suggestion
	if(prob(min(volume * 3), 100))
```

Reply:
@TheFakeElon What're you on about?

Reply:
@ike709 addresses your first issue :p

Reply:
Look at your code suggestion closer.

Reply:
AH

Reply:
Is the min even necessary? is a prob > 100 not guaranteed to pass?

Reply:
I remember seeing it once in like.. microwave code, although I don't think it's needed